### DISCUSSION TITLE
next.config redirects exclude 2 path parameters

### DISCUSSION THREAD

**Discussion by andrei9669** [vercel/next.js#26686] [Help]
in [this link](https://nextjs.org/docs/messages/invalid-route-source) it shows how to exclude a single value. but in my case, I need to exclude 2 or more.

I tried it like this:
```
{
    source: `/((?!article|category).*)/:id`,
    destination: `/article/:id`,
    permanent: true,
},
``` 
but it seems like it didn't work since the whole app just froze.

if I do just the category, then the page goes into an infinite loop.

any help is appreciated.

**Comment by wwwdepot** [comment]
I think you need to put `_next` in the negative group.

**Reply by andrei9669** [reply]
could you give an example, please? I'm not quite following what you mean.

**Reply by wwwdepot** [reply]
Exclude `_next` directory, so they don't get redirected. 
```jsonc
{
    source: `/((?!article|category|_next).*)/:id`,
    destination: `/article/:id`,
    permanent: true,
},
```

**Reply by andrei9669** [reply]
hey, sorry for the late reply, but unfortunately, it seems like it's not working.
